src/tasks/Common/PInvokeCollector.cs

```diff
@@ -0,0 +1,254 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can we leave this file in src/tasks/WasmAppBuilder?   I like that the code moved to its own file, but I don't think we want to reuse this in other tasks - we want a SRM-based pinvoke collector in the future, not SRE.

Reply:
And put the SignatureMapper back in there, too

Reply:
Moved them back.